src/olympia/addons/tasks.py

```diff
@@ -563,3 +564,26 @@ def flag_high_hotness_according_to_review_tier():
     NeedsHumanReview.set_on_addons_latest_signed_versions(
         qs, NeedsHumanReview.REASONS.HOTNESS_THRESHOLD
     )
+
+
+ERRONEOUSLY_ADDED_OVERGROWTH_DATE_RANGE = (
+    date(2024, 11, 5),
+    date(2024, 11, 7),
+)
+
+
+@task
+@use_primary_db
+def delete_erroneously_added_overgrowth_needshumanreview(addon_ids, **kw):
+    addons = Addon.unfiltered.filter(pk__in=addon_ids).no_transforms()
+    for addon in addons:
+        log.info(
+            'Deleting erroneously added NHR and updating due dates for %s', addon.pk
+        )
+        NeedsHumanReview.objects.filter(
+            version__addon=addon,
+            reason=NeedsHumanReview.REASONS.HOTNESS_THRESHOLD,
+            created__range=ERRONEOUSLY_ADDED_OVERGROWTH_DATE_RANGE,
+            is_active=True,
```

Comment:
I'm not a fan of how we have to query on the same things in two places but I can't immediately think of a way around it 😒

Reply:
Same. I could have generated some dictionaries of filters with an optional `versions__` prefix but I figured this wasn't worth doing for a one-off.